pages/Christmas-Items.md

```diff
@@ -0,0 +1,48 @@
+Christmas items are [Christmas specialities](https://github.com/Slimefun/Slimefun4/wiki/Christmas-Seasonal-Category).<br>
+These items consist of mostly foods.
+
+| Food | Ingredients | Workbench | Output Amount |
+| ---- | ----------- | --------- | ------------- |
+| Christmas Milk | 1 Bucket of Milk, 1 Glass Bottle | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 4 |
+| Christmas Chocolate Milk | 1 Christmas Milk, 1 Cocoa Beans | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+| Christmas Egg Nog | 1 Christmas Milk, 1 Egg | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+| Christmas Apple Cider | 1 Apple Juice, 1 Sugar | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+| Christmas Cookie | 1 Cookie, 1 Sugar, 1 Lime Dye | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 16 |
+| Christmas Fruit Cake | 1 Egg, 1 Apple, 1 Melon, 1 Sugar | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 4 |
+| Christmas Apple Pie | 1 Sugar, 1 Apple, 1 Egg | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+| Christmas Hot Chocolate | 1 Christmas Chocolate Milk | [Smeltery](https://github.com/Slimefun/Slimefun4/wiki/Smeltery) | 1 |
+| Christmas Cake | 1 Sugar, 1 Egg, 1 Wheat Flour, 1 Bucket of Milk | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 4 |
+| Christmas Caramel | 2 Sugars | [Smeltery](https://github.com/Slimefun/Slimefun4/wiki/Smeltery) | 4 |
+| Christmas Caramel Apple | 1 Christmas Caramel, 1 Apple, 1 Stick | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+| Christmas Chocolate Apple | 1 Cocoa Beans, 1 Apple, 1 Stick | [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table) | 2 |
+
+
+## Christmas Present
+The Christmas Present is even more special. Because it has the Santa's Magic!<br>
+When you open 1 of these, you can earn a random gift.
```

Comment:
```suggestion
When you open 1 of these, you obtain a random gift.
```
The usage of can here imposes that there is also the possibility of not receiving anything at all.
Unless that is actually the case, I suggest removing the can from that sentence